blame.php

```diff
@@ -33,7 +33,7 @@
 // Make sure that we have a repository
 if (!$rep)
 {
-	renderTemplateNoRepo('blame');
+	renderTemplate404('blame',$lang['NOREP']);
```

Comment:
You should just forward the key `NOREP` and let `renderTemplate404` handle the lookup in `$lang`. This saves typing by the caller and at some point we might not associate an error message only with that key, like is the case currently, but e.g. a special template as well. You already made `$lang` available in `renderTemplate404` anyway, but most likely simply forgot to use it.

    renderTemplate404('blame', 'NOREP');

And:

    $vars['error'] = "WebSVN 404: " . $lang[$errorDetail];

Reply:
ahh.. yes yes... this is my bad... I am fixing this...

Reply:
Done changed this.